src/main/java/com/google/cloud/datafusion/plugin/ariba/source/util/ResourceText.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.datafusion.plugin.ariba.source.util;
+
+import java.text.MessageFormat;
+import java.util.Locale;
+import java.util.ResourceBundle;
+import javax.annotation.Nullable;
+
+/**
+ * Util class to load and use the localized resource bundles for text and
+ * messages.
+ *
+ * @author dhirajkumar
```

Comment:
We usually don't add author in plugin code.

Reply:
Removed